docs/rest_api.md

```diff
@@ -2,3 +2,266 @@
+
+Let's create a trigger with name `periodic` to be fire at a specified frequency, every 2 hours (i.e. 02:00:00, 04:00:00, ...).
+
+Using the CLI this will be done with one command
+```bash
+wsk trigger create periodic --feed /whisk.system/alarms/alarm \
+  --param cron "0 */2 * * *" -v
+```
+As you will see because we are using the `-v` flag is that two HTTP requests are sent, one is to create a trigger `periodic` and the other is to invoke a feed action `/whisk.system/alarms/alarm` with the parameters to configure the feed provider to fire the trigger every 2 hours.
+
+To do the same with the REST API, lets create the trigger first
+```bash
+curl -u $AUTH -d '{"name":"periodic","annotations":[{"key":"feed","value":"/whisk.system/alarms/alarm"}]}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/periodic?overwrite=true 
+```
+
+As you can see the annotation `feed` is stored in the trigger which later we are going to use to know which feed action to use when deleting the trigger. 
+
+Now that the trigger is created, lets invoke the feed action
+```bash
+curl -u $AUTH -d "{\"authKey\":\"$AUTH\",\"cron\":\"0 */2 * * *\",\"lifecycleEvent\":\"CREATE\",\"triggerName\":\"/_/periodic\"}" -X POST -H "Content-Type: application/json" "https://openwhisk.ng.bluemix.net/api/v1/namespaces/whisk.system/actions/alarms/alarm?blocking=true&result=false"
+```
+
+Deleting the trigger, it's a similar process as creating the trigger, this time deleting the trigger and also using the feed action to configure the feed provider to delete the handler for the trigger.
+
+Invoke the feed action to delete the trigger handler from the feed provider
+```bash
+curl -u $AUTH -d "{\"authKey\":\"$AUTH\",\"lifecycleEvent\":\"DELETE\",\"triggerName\":\"/_/periodic\"}" -X POST -H "Content-Type: application/json" "https://openwhisk.ng.bluemix.net/api/v1/namespaces/whisk.system/actions/alarms/alarm?blocking=true&result=false"
+```
+
+Now delete the trigger with a HTTP request using `DELETE` method
+```bash
+curl -u $AUTH -X DELETE -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/triggers/periodic
+```
+
+## Rules
+
+To create a rule that associates a trigger with an action, send a HTTP request with a `PUT` method providing the trigger and action in the body of the request.
+```bash
+curl -u $AUTH -d '{"name":"t2a","status":"","trigger":"/_/events","action":"/_/hello"}' -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/rules/t2a?overwrite=true
+```
+
+Rules can be enable or disable, you can change the status of the rule by updating its status property. For example, to disable the rule `t2a` send in the body of the request `status:inactive` with a `POST` method.
```

Comment:
Rules can be enable or disable, you can -> Rules can be enabled or disabled, and you can